docs/website/netcdf-java/ncml/index.htm

```diff
@@ -38,7 +38,7 @@ <h3>NcML as output </h3>
 <p>The simplest use of NcML is to describe the metadata and structural content of a netCDF file, like the output of <strong>ncdump -h</strong>. There are
   several ways to generate NcML ouput from an existing netCDF file:</p>
 <ol>
-  <li>In the <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a> application, use the NcML tab to open a
+  <li>In the <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a> application, use the NcML tab to open a
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.